src/tools/options.cpp

```diff
@@ -132,6 +132,8 @@ bool tools::parseConfigurationParameter(Configuration& config, const std::string
                 config.additionalOptions.replaceNopThreshold = intValue;
             else if(paramName == "register-resolver-rounds")
                 config.additionalOptions.registerResolverMaxRounds = intValue;
+            else if(paramName == "extract-loads-from-loops")
+                config.additionalOptions.moveConstantsDepth = Optional<int>(intValue);
```

Comment:
Adding the OptimizationPass with the parameter-name `extract.loads-from-loops` (see Optimizer.cpp) will enable/disable the optimization.
The separate parameter to set the number of nested loops to handle will need to have a different parameter-name!